tests/Integration/goldens/talent/src/V4beta1/Client/BaseClient/JobServiceBaseClient.php

```diff
@@ -61,6 +61,17 @@
  * contained within formatted names that are returned by the API.
  *
  * @experimental
+ *
+ * @method GuzzleHttp\Promise\PromiseInterface batchCreateJobsAsync(\Google\Cloud\Talent\V4beta1\BatchCreateJobsRequest $request, array $optionalArgs = [])
+ * @method GuzzleHttp\Promise\PromiseInterface batchDeleteJobsAsync(\Google\Cloud\Talent\V4beta1\BatchDeleteJobsRequest $request, array $optionalArgs = [])
+ * @method GuzzleHttp\Promise\PromiseInterface batchUpdateJobsAsync(\Google\Cloud\Talent\V4beta1\BatchUpdateJobsRequest $request, array $optionalArgs = [])
+ * @method GuzzleHttp\Promise\PromiseInterface createJobAsync(\Google\Cloud\Talent\V4beta1\CreateJobRequest $request, array $optionalArgs = [])
+ * @method GuzzleHttp\Promise\PromiseInterface deleteJobAsync(\Google\Cloud\Talent\V4beta1\DeleteJobRequest $request, array $optionalArgs = [])
+ * @method GuzzleHttp\Promise\PromiseInterface getJobAsync(\Google\Cloud\Talent\V4beta1\GetJobRequest $request, array $optionalArgs = [])
+ * @method GuzzleHttp\Promise\PromiseInterface listJobsAsync(\Google\Cloud\Talent\V4beta1\ListJobsRequest $request, array $optionalArgs = [])
+ * @method GuzzleHttp\Promise\PromiseInterface searchJobsAsync(\Google\Cloud\Talent\V4beta1\SearchJobsRequest $request, array $optionalArgs = [])
+ * @method GuzzleHttp\Promise\PromiseInterface searchJobsForAlertAsync(\Google\Cloud\Talent\V4beta1\SearchJobsRequest $request, array $optionalArgs = [])
+ * @method GuzzleHttp\Promise\PromiseInterface updateJobAsync(\Google\Cloud\Talent\V4beta1\UpdateJobRequest $request, array $optionalArgs = [])
```

Comment:
1. I know that normally PHPDoc understands the typehints as PHP itself would, meaning the above `GuzzleHttp\Promise\PromiseInterface` would be interpreted as `\Google\Cloud\Talent\V4beta1\Client\BaseClient\GuzzleHttp\Promise\PromiseInterface`. We could either add a proceeding `\`, or add a `use GuzzleHttp\Promise\PromiseInterface;` at the top. I am okay with either way.
2. Given #&#x2060;1, we could shorten all the `$request` typehints to their base name, since they'll always be already imported for the sync methods.

Reply:
Soooo much cleaner. Done!

Reply:
wow, that really did make a huge difference